buildSrc/src/test/kotlin/BuildNumberComparisonTest.kt

```diff
@@ -2,23 +2,19 @@ import com.virtuslab.gitmachete.buildsrc.UpdateEapBuildNumber.Companion.buildNum
 import org.junit.Test
 
 class BuildNumberComparisonTest {
```

Comment:
Is this test triggered anywhere in our current CI pipeline?

Reply:
AFAIC gradle runs all things marked with @Test during build task.

Reply:
Okay, so if you fail a test in buildSrc on purpose, the entire build is broken? that's what really matters here (so that we don't overlook a failure)

Reply:
Yes, the build task fails.